File: javascript/problem006.js
var sum_of_the_square = 0
var root_square_of_the_sum = 0

for(i=1; i<=100; i++){
  sum_of_the_square += i * i;
  root_square_of_the_sum += i;
}
var result = root_square_of_the_sum * root_square_of_the_sum - sum_of_the_square;
console.log(result)

File: java/src/Problem006.java
/**
 * Created by 김 on 2016-02-05.
 */
public class Problem006 {


    int result() {
        int value1 = 0;
        int value2 = 0;
        for (int i = 1; i <= 100; i++) {
            value1 = value1 + i * i;
            value2 = value2 + i;
        }
        value2 = value2 * value2;
        return value2 - value1;
    }

    public static void main(String args[]) {
        Problem006 problem = new Problem006();
        System.out.println(problem.result());
    }
}
